### DISCUSSION TITLE
Table Row User Ownership

### DISCUSSION THREAD

**Discussion by savannt** [Budibase/budibase#8149] [Help] [answered]
Hello guys!

I was curious what's the best way is to implement table rows that can be "owned" by a certain user?

My current solution is to setup a relationship column pointing to the User to denote ownership.
This works great, but when using this in tandem with multiple Relationships the Relationship Picker ignores the User relationship on select dropdown and shows all rows.

**Comment by melohagan** [comment, marked as answer]
Hey @B3N909 

There's work to be done around role-based access to table rows: https://github.com/Budibase/budibase/issues/4789

Your idea was great, but we would need filtering to be added to relationship pickers: https://github.com/Budibase/budibase/issues/7810
